src/MainWindow.vala

```diff
@@ -40,7 +40,7 @@ public class Installer.MainWindow : Gtk.Dialog {
             deletable: false,
             height_request: 700,
             icon_name: "system-os-installer",
-            resizable: false,
+            resizable: true,
```

Comment:
I remember this fixed a specific bug, but I don't recall if it was pop-specific due to the artwork or not. @danrabbit thoughts on making the installer resizable?

Reply:
The issue was for disks that had many partitions -- the window is too small to comfortably accommodate disks with more than a few partitions.